Formula/fn.rb

```diff
@@ -27,7 +27,6 @@ def install
 
   test do
     require "socket"
-    assert_match version.to_s, shell_output("#{bin}/fn --version")
```

Comment:
Removed due to upstream forget to update their [hardcoded](https://github.com/fnproject/cli/blob/706f4031c48f9dc55b912a0845fe63bf1c170260/config/version.go#L13) version number.

Reply:
Probably worth `inreplace`-ing, or seeing if upstream want to fix it & cut a new release.

Reply:
`inreplace`-d, and reported upstream as fnproject/cli#376.

Reply:
Strange that there's also a version.go file in the root directory which contains the correct version number.